database/init.sql

```diff
@@ -1,7 +1,21 @@
 CREATE TABLE account (
   id SERIAL PRIMARY KEY,
   email TEXT UNIQUE NOT NULL,
-  password TEXT NOT NULL
```

Comment:
This shouldn't be removed

Reply:
fixed already, after testing I copied DB script from pgadmin and removed accidentally